src/core/runtime.hpp

```diff
@@ -382,6 +378,44 @@ struct SHMesh : public std::enable_shared_from_this<SHMesh> {
 
   ~SHMesh() { terminate(); }
 
+  void preCompose(const std::shared_ptr<SHWire> &wire, SHVar input = shards::Var::Empty) {
```

Comment:
Why is it called preCompose? It seems to call composeWire, so what's different?

Reply:
Can't it be just called composeWire or something

Reply:
Because it implies you pass false to schedule (compose parameter).
But you are right, can just be "compose" actually.